cmd/handler/main.go

```diff
@@ -263,8 +263,7 @@ func createHealthyFile() error {
 
 func checkNmstateIsWorking() error {
 	setupLog.Info("Checking availability of nmstatectl")
-	_, err := nmstatectl.Show()
-	if err != nil {
+	if err := nmstatectl.ShowWithArgumentsAndOutputs([]string{"-vvv"}, os.Stdout, os.Stderr); err != nil {
```

Comment:
This part worries me because it will be printed as a raw output of bash `nmstatectl show -vvv` inside the handler pod logs.

The log has already a parseable structure. If we add this, it means you cannot any more parse `oc logs nmstate-handler`. Good enough for debugging, but I am not sure that it's good for "ok" runs

Reply:
Integrated with logger at latest push.